porcupine/plugins/run/dialog.py

```diff
@@ -105,8 +105,14 @@ def __init__(self, file_path: Path, project_path: Path, suggestions: list[histor
             validated_callback=self.update_run_button,
         )
 
-        sub_text = "\n".join("{%s} = %s" % pair for pair in substitutions.items())
-        ttk.Label(content_frame, text=f"Substitutions:\n{sub_text}\n").pack(fill="x")
```

Comment:
With a label, the width of the dialog is determined by the longest substitution text, or the width of "Display the output inside the Porcupine window (does not support keyboard input)", whichever is wider. This might be a good thing, because if you have very long file paths, then the dialog will be bigger, and they will always fit.

On the other hand, as you say, a text widget will hopefully allow copy/pasting in the future.

Which is better overall? What do you think?

Reply:
If the user has that long file paths, he deserves to resize the window to see it properly. So I vote for the text widget.